sensor/common/scan/metrics/metrics.go

```diff
@@ -0,0 +1,26 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/stackrox/rox/pkg/metrics"
+)
+
+var (
+	SensorScanSemaphoreQueueSize = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.SensorSubsystem.String(),
+		Name:      "sensor_scan_semaphore_queue_size",
```

Comment:
Same comment about the label here: `location=sensor-scan`, although I ask myself what is "sensor-scan"? Maybe a better name here would be useful.

Reply:
I went with longer `sensor-local-image-scan`